tests/by-util/test_cp.rs

```diff
@@ -3701,3 +3703,57 @@ fn test_cp_no_such() {
         .fails()
         .stderr_is("cp: 'no-such/' is not a directory\n");
 }
+
+#[cfg(all(unix, not(target_os = "macos")))]
+fn compare_xattrs<P: AsRef<Path>>(path1: P, path2: P) -> bool {
```

Comment:
i will update this code to use the one in
https://github.com/uutils/coreutils/pull/5835

after this one is merged